eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -49,16 +48,19 @@
                 <source>..\..\sdk\core\azure-core-http-okhttp\src\samples\java\com</source>
                 <source>..\..\sdk\core\azure-core-management\src\main\java\com</source>
                 <source>..\..\sdk\core\azure-core-test\src\main\java\com</source>
+                <source>..\..\sdk\core\azure-core-tracing-opencensus\src\main\java\com</source>
                 <source>..\..\sdk\eventhubs\azure-messaging-eventhubs\src\main\java\com</source>
                 <source>..\..\sdk\eventhubs\azure-messaging-eventhubs\src\samples\java\com</source>
                 <source>..\..\sdk\eventhubs\azure-messaging-eventhubs-checkpointstore-blob\src\main\java\com</source>
                 <source>..\..\sdk\eventhubs\azure-messaging-eventhubs-checkpointstore-blob\src\samples\java\com</source>
                 <source>..\..\sdk\identity\azure-identity\src\main\java\com</source>
                 <source>..\..\sdk\identity\azure-identity\src\samples\java\com</source>
-                <!--                <source>..\..\sdk\keyvault\azure-keyvault-keys\src\main\java\com</source>-->
-                <!--                <source>..\..\sdk\keyvault\azure-keyvault-keys\src\samples\java\com</source>-->
-                <!--                <source>..\..\sdk\keyvault\azure-keyvault-secrets\src\main\java\com</source>-->
-                <!--                <source>..\..\sdk\keyvault\azure-keyvault-secrets\src\samples\java\com</source>-->
+<!--                <source>..\..\sdk\keyvault\azure-security-keyvault-certificates\src\main\java\com</source>-->
+<!--                <source>..\..\sdk\keyvault\azure-security-keyvault-certificates\src\samples\java\com</source>-->
+                <source>..\..\sdk\keyvault\azure-security-keyvault-keys\src\main\java\com</source>
+<!--                <source>..\..\sdk\keyvault\azure-security-keyvault-keys\src\samples\java\com</source>-->
```

Comment:
Can the samples be uncommented too for Keys and Secrets?

Reply:
They currently have spotbugs issues. @g2vinay will be resolving this soon.